core/jobs/batch_jobs/blog_validation_jobs.py

```diff
@@ -113,10 +113,11 @@ def run(
         )
 
 
-# TODO(#15613): Due to incomplete typing of apache_beam library and absences
-# of stubs in Typeshed, MyPy assuming PTransform class is of type Any. Thus to
-# avoid MyPy's error (Class cannot subclass 'PTransform' (has type 'Any')) , we
-# added an ignore here.
+# TODO(#15613): Here we use MyPy ignore because the incomplete typing of
```

Comment:
Can we add just the stubs we need (here and elsewhere)?

Reply:
https://github.com/oppia/oppia/pull/15501#discussion_r887164703

Currently, there are no stubs for the beam. Also, the library is complex and very big -- so if we add stubs by ourselves then it will take much much time because this is equivalent to reading through a whole new codebase and writing the stubs for it. Also, we are using `|`  operator in the codebase which is not legal according to python 3 -- this is also a hurdle in stubs.